samples/generate-mock-data.json

```diff
@@ -1,7 +1,7 @@
 {
   "uuid": "",
   "lastMigration": 13,
-  "name": "Tutorial - Generate mock data",
+  "name": "Mock-Data",
```

Comment:
I would rather keep the original title as this mock is linked to the "Generate mock data" [tutorial](https://mockoon.com/tutorials/generate-mock-json-data/) of the website.

Reply:
Yes sure. Changes updated.